packages/storybook/src/nimble/patterns/anchor/anchor-docs.ts

```diff
@@ -0,0 +1,5 @@
+export const hrefDescription = (componentName: string, includeDisable: boolean): string => `The URL that the ${componentName} points to. 
```

Comment:
instead of boolean flags can you change this to an options object so it's easier to intuit what is being configured at the call site?

```
hrefDescription('anchor tree item', false)
```
vs

```
hrefDescription({ tagName: 'anchor tree item', includeDisableInstructions: false })
```

Reply:
Good idea. I applied that pattern to the descriptions in `storybook.ts` too, just in case we find we need to parameterize them as we use them in other stories.